Swift/BRCryptoDemo/WalletViewController.swift

```diff
@@ -117,12 +117,6 @@ class WalletViewController: UITableViewController, TransferListener {
                     self.tableView.reloadRows(at: [path], with: .automatic)
                 }
 
-            case .confirmation:
-                if let index = self.transfers.firstIndex (of: transfer) {
-                    let path = IndexPath (row: index, section: 0)
-                    self.tableView.reloadRows(at: [path], with: .automatic)
-                }
-
```

Comment:
And here too.  The `reloadRows` will change the color indicator from 'yellow' (pending), to 'green' (confirmed/included).

Reply:
Updated to account for this.